go.mod

```diff
@@ -5,8 +5,8 @@ go 1.18
 require (
 	github.com/cosmos/cosmos-sdk v0.45.11
 	github.com/cosmos/go-bip39 v1.0.0
-	github.com/cosmos/ibc-go/v3 v3.4.0
-	github.com/cosmos/interchain-security v1.0.0-rc2
+	github.com/cosmos/ibc-go/v4 v4.2.0
+	github.com/cosmos/interchain-security v1.0.0-rc2.0.20230120130136-d86a63f44f5d
```

Comment:
Why not ICS v1.0.0-rc3?

Reply:
Ah I did originally set it, I'm not sure why this happened